src/components/Label/Label.test.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { render, screen, axe } from 'testUtils';
+
+import { Input, Box } from '../';
+import Label from './Label';
+
+test('renders correctly', () => {
+  render(
+    <Box>
+      <Label id="username" data-testid="my-label">
+        Hello world
+      </Label>
+      <Input aria-labelledby="username" />
+    </Box>,
+  );
+
+  const label = screen.getByTestId('my-label');
+
+  expect(label).toBeInTheDocument();
+  expect(label).not.toHaveStyleRule('height', '1px');
+  expect(label).not.toHaveStyleRule('position', 'absolute');
+});
+
+test('should hide label visually when given hidden prop', () => {
```

Comment:
```suggestion
test('hides label visually when hidden is true', () => {
```
just to be more specific - `given hidden prop` could also apply for `hidden={false}`?

Reply:
have made test title more specific